CMakeLists.txt

```diff
@@ -79,6 +79,12 @@ option (SEQAN_ARCH_AVX2 "Build AVX optimized binaries." OFF)
 option (SEQAN_ARCH_NATIVE "Build natively optimized binaries." OFF)
 
 # ===========================================================================
+# Version Check Variable.
+# ===========================================================================
+
+option (SEQAN_VERSION_CHECK "SeqAn version check." ON)
```

Comment:
Nop, this is correct. We keep it in as compile time option, but the default of the version-check option is set to OFF. This means the people get informed, that there is such an option, when they run the help, but they have to opt-in explicitly and won't be affected if they don't.